frontend/awx/resources/inventories/inventoryHostsPage/InventoryHostFacts.tsx

```diff
@@ -15,7 +15,13 @@ export function InventoryHostFacts(props: { page: string }) {
 
   return (
     <PageDetails>
-      <PageDetailCodeEditor label={t('Facts')} value={JSON.stringify(facts) || '{}'} />
+      <PageDetailCodeEditor
+        label={t('Facts')}
+        value={JSON.stringify(facts) || '{}'}
+        fullWidth
+        toggleLanguage
```

Comment:
We should make toggleLanguage and showCopyToClipboard defaulted to on.

Reply:
Pushed updates